core_impl/src/test/java/io/opencensus/impl/tags/TagContextImplTest.java

```diff
@@ -86,6 +89,19 @@ public void testClear() {
   }
 
   @Test
+  public void testIterator() {
+    TagContext tags = tagContexts.emptyBuilder().set(KS1, V1).set(KS2, V2).build();
+    Iterator<Tag> i = tags.unsafeGetIterator();
+    assertTrue(i.hasNext());
+    assertThat(i.next()).isEqualTo(TagString.create(KS1, V1));
```

Comment:
The order of elements to be iterated is non-deterministic, similar to https://github.com/census-instrumentation/opencensus-java/issues/224.